compile-all

```diff
@@ -161,12 +179,12 @@ fi
 
 # C flags for GCC, MINGW64
 if [ "$CC" == "gcc" ] ; then
-   export CFLAGS="-I/usr/local/include -fstrict-aliasing -fno-omit-frame-pointer"
+   export CFLAGS="-I${BASE_USR_DIR}/usr/local/include -fstrict-aliasing -fno-omit-frame-pointer"
```

Comment:
Since BASE_USR_DIR ends in a / should it be this instead?
${BASE_USR_DIR}usr/local/include

Reply:
I kept the "/" on both ends because multiple slashes in a row in a file-path gets simplified down to a single slash. So having something like "/prefix//usr/local/include" is fine. This is actually more error-resistant than trying to remember which end of the prefix the slash is supposed to be on.

Reply:
Looks fine to me too. Sorry I didn't look sooner